packages/labextension/package.json

```diff
@@ -30,9 +30,9 @@
   "scripts": {
     "build": "npm run build:lib && npm run build:labextension",
     "build:dev": "npm run build:lib",
-    "build:labextension": "rimraf dist && mkdirp dist && cd dist && npm pack ..",
+    "build:labextension": "rimraf dist && mkdirp dist && jupyter labextension build . && cd dist && npm pack ..",
```

Comment:
The `jupyter` command requires the python jupyterlab package to be available when the command is run. To ensure this we would probably need to use PEP517 build/install flow.

Reply:
Thanks, I added a `pyproject.toml` file.